docs/v1/P4Runtime-Spec.mdk

```diff
@@ -658,17 +669,16 @@ follows:
   different (`device_id`, `role_id`) pairs; it allows it.
 
 * To start a controller session, a controller first opens a bidirectional stream
```

Comment:
I feel we agreed that is also needed if the controller wishes to mutate state (via one of the P4RT RPCs) down the road, and a stream-channel is not required for just reading state. Could you please disambiguate this?

Reply:
Done.